extensions/EntityExtension/src/main/kotlin/com/typewritermc/entity/entries/activity/PlayerCloseByActivity.kt

```diff
@@ -63,12 +75,14 @@ class PlayerCloseByActivityEntry(
 
 class PlayerCloseByActivity(
     private val range: Double,
+    private val activationRange: Optional<Double>,
```

Comment:
Instead of having this be optional, why not always have it be a boolean, and when not provided, just have it be equal to the range? It would simplify the code below as well.